include/oneapi/dpl/pstl/hetero/algorithm_impl_hetero.h

```diff
@@ -711,6 +711,11 @@ __pattern_find_if(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _Iterato
 // find_end
 //------------------------------------------------------------------------
 
+template <typename Name>
+class equal_wrapper
```

Comment:
As example, lets take a look to 
```C++
template <typename _Name>
struct __walk_brick_wrapper;

template <typename _BackendTag, typename _ExecutionPolicy, typename _ForwardIterator, typename _Function>
void
__pattern_walk_brick(__hetero_tag<_BackendTag> __tag, _ExecutionPolicy&& __exec, _ForwardIterator __first,
                     _ForwardIterator __last, _Function __f)
{
    if (__last - __first <= 0)
        return;

    __pattern_walk1(
        __tag,
        __par_backend_hetero::make_wrapped_policy<__walk_brick_wrapper>(::std::forward<_ExecutionPolicy>(__exec)),
        __first, __last, __f);
}
```

I think better continue use `struct` instead `class` and do not use body at all.

Reply:
yes, lets remove a empty body  - `{}`